package.json

```diff
@@ -13,13 +13,20 @@
   "devDependencies": {
     "@astrojs/mdx": "^0.10.0",
     "@astrojs/react": "^1.0.0",
+    "@astrojs/renderer-react": "^0.5.0",
     "@astrojs/rss": "^1.0.0",
     "@astrojs/sitemap": "^1.0.0",
     "astro": "^1.0.6",
     "prettier": "^2.7.1",
     "prettier-plugin-astro": "^0.5.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "@types/react": "18.0.17",
     "sass": "^1.54.5"
+  },
+  "dependencies": {
+    "@emotion/react": "^11.10.0",
```

Comment:
Remove emotion since it's not being used now

Reply:
Done. Check now

Reply:
Looks good